### DISCUSSION TITLE
How to change aspect of LNK folders?

### DISCUSSION THREAD

**Discussion by QQ7ita** [doublecmd/doublecmd#1994] [General]
DC v.1.1.20-607
is there any way to change the appearance or color of symbolic link folders?
tnx
Flavio

**Comment by alexx2000** [comment]
https://doublecmd.github.io/doc/en/configuration.html#ConfigColorFiles

**Reply by QQ7ita** [reply]
I have tried but without success, I have setted:
Category Name: LNK
Category Mask: *
Category attributes: la
Category color: some color

**Reply by alexx2000** [reply]
`Category attributes: l*`

But it will work with any symbolic links, for directories only you need to use a search template.
